2-mandatory/4-people-i-know.js

```diff
@@ -394,6 +401,11 @@ This time, I only want the full names ("<firstname> <lastname>") of my friends w
 */
 
 let friendsWhoAreColleaguesOfStacie = [];
+for (const myFriends in friends) {
+  if( friends[myFriends].colleagues.map(x=>(x.name).include("Stacie Villarreal") )){
```

Comment:
Why include() anf not equals?

This is a bit hard to read. A forEach loop is simpler